### DISCUSSION TITLE
sync the dir of the current to the one in the built in console

### DISCUSSION THREAD

**Discussion by jmlucjav** [doublecmd/doublecmd#1612] [Suggestions]
hi,

Im starting to use DC, so not sure if this is possible...:

- I am in dir X, and I want to change to some very distant dir...
- I open the built in console
- I cd to the dir I want (or use a tool like zoxide that allows me to reach that dir pretty fast)
- now I want DC to change to this dir, when I close the console

is this possible? it is supper handy cause zoxcide can reach many dirs easily, very fast (that I wont have in my hotlist)

thanks

**Comment by alexx2000** [comment]
Hi, you are already got a right [answer](https://doublecmd.h1n.ru/viewtopic.php?p=43710#p43710), so you can use something like this:

`doublecmd -c $(pwd) > /dev/null`

No other way exists.

**Comment by jmlucjav** [comment]
ok...I didn't understood that answer, now I see the -c param and I see...not the most convinient way but it's just one additional command, easy with an alias...thanks!!
